Sources/ValidatingProperty.swift

```diff
@@ -178,26 +178,6 @@ public final class ValidatingProperty<Value, ValidationError: Swift.Error>: Muta
 		self.init(MutableProperty(initial), with: other, validator)
 	}
 
-	/// Create a `ValidatingProperty` that presents a mutable validating
-	/// view for an inner mutable property.
-	///
-	/// The proposed value is only committed when `valid` is returned by the
-	/// `validator` closure.
-	///
-	/// - note: `inner` is retained by the created property.
-	///
-	/// - parameters:
-	///   - inner: The inner property which validated values are committed to.
-	///   - other: The property that `validator` depends on.
-	///   - validator: The closure to invoke for any proposed value to `self`.
-	public convenience init<U, E>(
```

Comment:
This changes the semantic. [This overload](https://github.com/ReactiveCocoa/ReactiveSwift/pull/644/files/e27310de692416a4fd70edfd1896ff3245ca4b51#diff-42b514003f72de821971eab375eda51dR192) was supposed to be the primary one, and the removed one here should have been implemented in terms of that.

Reply:
So what should be done here to avoid the compilation error? @andersio

Reply:
Feel free pushing your changes on this branch.